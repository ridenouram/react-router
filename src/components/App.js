import React from 'react';
import {
  BrowserRouter as Router,
  Route,
  Switch,
} from 'react-router-dom';
// import Red from './Red';
// import Yellow from './Yellow';
// import Blue from './Blue';
import Header from './Header';
import Color from './Color';

export default function App() {
  return (
    <Router>
      <Header />
      <Switch>
        {/* <Route path="/red" component={Red} />
        <Route path="/yellow" component={Yellow} />
        <Route path="/blue" component={Blue} /> */}
        <Route path="/color/:id" component={Color} />
      </Switch>
    </Router>
  );
}
