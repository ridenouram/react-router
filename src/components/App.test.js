import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});

