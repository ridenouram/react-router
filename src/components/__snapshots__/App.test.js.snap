// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<BrowserRouter>
  <Header />
  <Switch>
    <Route
      component={[Function]}
      path="/red"
    />
    <Route
      component={[Function]}
      path="/yellow"
    />
    <Route
      component={[Function]}
      path="/blue"
    />
  </Switch>
</BrowserRouter>
`;
